book/en/src/preface.md

```diff
@@ -21,6 +21,18 @@ This is the documentation of v1.0.x of RTIC; for the documentation of version
 * v0.5.x go [here](/0.5).
 * v0.4.x go [here](/0.4).
 
+## Is RTIC an RTOS?
+
+A common question is wether RTIC is an RTOS or not, and depening on your background the
+answer may vary. From RTIC's developers the view is that RTIC is an hardware accelerated
+RTOS that utilizes the NVIC in Cortex-M MCUs to perform scheduling, rather than the more
+classical software kernel.
+
+Another common view from the community is that RTIC is an concurrency toolkit as there
```

Comment:
```suggestion
Another common view from the community is that RTIC is a concurrency framework as there
```

Reply:
To match our tagline: "A concurrency framework for building real-time systems" rtic.rs